script/maximum-living-area-drop-down.js

```diff
@@ -0,0 +1,26 @@
+function maximumLivingAreaDropDown() {
```

Comment:
Rewrite function to [arrow function expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)